inc/csp.php

```diff
@@ -155,6 +156,13 @@ function allow_wikimedia_origins( array $allowed_origins, string $policy_type ):
 	if ( in_array( $policy_type, [ 'script-src', 'style-src', 'img-src' ], true ) ) {
 		$allowed_origins[] = 'https://*.wikimedia.org';
```

Comment:
I found an error loading an image from Wikipedia.

```suggestion
		$allowed_origins[] = 'https://*.wikimedia.org';
		$allowed_origins[] = 'https://wikipedia.org';
```